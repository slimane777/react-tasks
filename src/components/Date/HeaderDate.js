import React from 'react'
import moment from 'moment'
import { Wrapper, Day, Month, Row } from './styles'

const HeaderDate = () => {
  return (
    <Wrapper>
      <Row>
        <Day weight={'600'}>
            {moment(new Date()).format('dddd,')}
        </Day>
        <Day weight={'400'}>
          {moment(new Date()).format(' Do')}
        </Day>
      </Row>
      <Month>
        {moment(new Date()).format('MMMM')}
      </Month>
    </Wrapper>
  )
}

export default HeaderDate;