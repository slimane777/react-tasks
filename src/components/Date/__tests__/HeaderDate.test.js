import React from 'react';
import renderer from 'react-test-renderer';
import HeaderDate from '../HeaderDate';
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'

describe('HeaderDate test', () => {
  const initialState = {tasks:{tasks: []}}
  const mockStore = configureStore()
  let store

  it('HeaderDate should match snapshot', () => {
    store = mockStore(initialState)
    const component = renderer.create(<Provider store={store}><HeaderDate
      /></Provider>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
