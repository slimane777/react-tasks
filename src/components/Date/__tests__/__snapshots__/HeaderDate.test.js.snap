// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderDate test HeaderDate should match snapshot 1`] = `
<div
  className="sc-hKFxyN eUBRzi"
>
  <div
    className="sc-gtsrHT dhDFSK"
  >
    <p
      className="sc-bdnxRM eEyHpk"
    >
      Monday,
    </p>
    <p
      className="sc-bdnxRM jORmTa"
    >
       12th
    </p>
  </div>
  <p
    className="sc-dlnjwi bkiDZf"
  >
    July
  </p>
</div>
`;
